samples/tutorial/Tutorial2.md

```diff
@@ -209,17 +147,17 @@ Finally, update `render` for `TodoListWorkflow` to send the titles of the todo m
 ```kotlin
 object TodoListWorkflow : StatefulWorkflow<Unit, State, Nothing, TodoListScreen>() {
 
+  // …
```

Comment:
I think it's missing the state here with the new todos

Reply:
I don't think so? We already declared `State.todos`, above and set up a cutesy placeholder TodoModel in `initialState`. Am I missing something?

Reply:
It's there my bad, github had hidden that line since it never changed all good!